configs/forseti_conf.yaml.sample

```diff
@@ -26,8 +26,8 @@ global:
     # notification
     # Remove or comment out the email properties if you do not want Forseti to
     # send email notifications.
-
-    email_recipient: EMAIL_RECIPIENT
+    # Multiple recipients can be specified as comma-separated text.
```

Comment:
I think the default value here is explanation enough of what's possible and the comment can go away.